defaults/main.yml

```diff
@@ -1 +1,7 @@
 ---
+
+sansible_amazon_inspector_agent_dest: /tmp/amazon_inspector_agent
+sansible_amazon_inspector_agent_download_only: no
```

Comment:
What's the use case for the download only flag?

Reply:
The agent only installs on EC2 instances.  Setting the flag allows to test the role on docker, Vagrant, etc.